sdk/keyvault/Azure.Security.KeyVault.Secrets/tests/SecretClientLiveTests.cs

```diff
@@ -53,7 +53,7 @@ public async Task SetSecret()
         public async Task SetSecretWithExtendedProps()
         {
             string secretName = Recording.GenerateId();
-            IResolveConstraint createdUpdatedConstraint = Is.EqualTo(DateTimeOffset.FromUnixTimeSeconds(1596061817));
+            IResolveConstraint createdUpdatedConstraint = Is.EqualTo(DateTimeOffset.FromUnixTimeSeconds(1612561237));
```

Comment:
what's this magic number? :)

Reply:
Lotto numbers. Or the current time from the first recording, which I then updated in the subsequent recording (in this case, sync ran first, then async, so I updated async manually). Probably should either stop asserting this (arguably going too far asserting it) or figure out a better way. I only need to do this when re-recording, which for secrets is only when new service versions roll out.